proc_scripts/load_data.py

```diff
@@ -95,43 +243,92 @@ def proc_Hahn_echoph(s, fl=None):
         fl.image(abs(s))
     return s
 
-def proc_spincore_IR(s,fl=None):
+def proc_spincore_IR(s,clock_correction,fl=None):
+    logger.info(strm("pre-processing for IR acquired on spincore"))
```

Comment:
do not include the clock correction as an argument here -- that's not part of the preprocessing.  This is especially true now that we have the algorithm that does the clock correction w/out any reference data.